# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xf86-video-via/xv.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- xf86-video-via-0.1.31.1_orig/src/via_video.c	2005-09-04 20:40:17.000000000 +0200
+++ xf86-video-via-0.1.31.1/src/via_video.c	2005-11-02 15:06:22.854356776 +0100
@@ -74,10 +74,6 @@
 #define  XV_IMAGE          0
 #define MAKE_ATOM(a) MakeAtom(a, sizeof(a) - 1, TRUE)
 
-#ifndef XvExtension
-void viaInitVideo(ScreenPtr pScreen) {}
-#else
-
 static vidCopyFunc viaFastVidCpy = NULL;
 
 
@@ -1207,4 +1203,3 @@
     pVia->swov.panning_y = y;
 }
 
-#endif  /* !XvExtension */
